tests/Integration/ApiPlatform/GetHookStatusTest.php

```diff
@@ -52,18 +52,18 @@ public function testGetHookStatus(): void
         $activeHook->add();
 
         $bearerToken = $this->getBearerToken();
-        $response = static::createClient()->request('GET', '/new-api/hook-status/' . (int) $inactiveHook->id, ['auth_bearer' => $bearerToken]);
+        $response = static::createClient()->request('GET', '/new-api/hook-status/' . (int) $inactiveHook->id, ['auth_bearer' => $bearerToken, 'headers' => ['accept' => ['application/json']]]);
```

Comment:
Why is it needed to add this everywhere? Shouldn't it be the default format (if we even keep the jsonld format)